recipe/meta.yaml

```diff
@@ -8,11 +8,13 @@ package:
 source:
   url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/opentelemetry_proto-{{ version }}.tar.gz
   sha256: 16286214e405c211fc774187f3e4bbb1351290b8dfb88e8948af209ce85b719e
+  patches:
+    - patches/0001-relax-protobuf.patch
```

Comment:
Of course, this will need removing again once the next upstream version is released, so I would understand if you wish to avoid that maintenance burden.

On the other hand, I'm happy to provide a further PR when that happens.

Reply:
Sure; seems pretty low traffic